modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -3,58 +3,74 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To ensure that your Mule applications always refer to a single source of truth for your API specifications, MuleSoft introduces a workflow to design, publish, and implement your API specifications.
+include::reuse::partial$beta-banner.adoc[tag="studio-asyncapi"]
+
+Ensure that your Mule applications always refer to a single source of truth for your API specifications. Design, publish, and implement your API specifications with the following workflow.
 
 . Design an API specification. +
-You can either xref:design-center::design-create-publish-api-visual-editor.adoc[create and publish an API specification with the Visual Editor] or xref:design-center::design-create-publish-api-raml-editor.adoc[with the Code Editor].
+You can either xref:design-center::design-create-publish-api-visual-editor.adoc[create and publish an API specification with the Visual Editor] or xref:design-center::design-create-publish-api-raml-editor.adoc[with the Text Editor].
 +
 You can also xref:create-api-specification-studio.adoc[create an API specification project in Studio and sync it with Design Center].
-. Publish that API specification to Exchange. +
+. Publish your API specification to Exchange. 
++
 Make your newly designed API available to your entire organization as an Exchange dependency. As you change your API, you can publish new versions of your API specifications to Exchange. See xref:exchange::to-change-raml-version.adoc#best-practices-api-versioning[best practices for API versioning].
 . Import the API specification from Exchange to Studio. +
-You can either xref:import-api-specification-exchange.adoc[import an API specification from Exchange] or xref:import-api-specification-maven.adoc[from Maven]. Studio imports your API specification as a dependency. This means that your Mule application project now holds a reference to the specific API version that you're implementing. Studio monitors your API and notifies you when there is a new version available.
+You can either xref:import-api-specification-exchange.adoc[import an API specification from Exchange] or xref:import-api-specification-maven.adoc[from Maven]. Studio imports your API specification as a dependency. This means that your Mule application project now holds a reference to the specific API version that you’re implementing. Studio checks your API and notifies you when there’s a new version available.
```

Comment:
```suggestion
You can import an API specification from either from xref:import-api-specification-exchange.adoc[Exchange] or xref:import-api-specification-maven.adoc[Maven]. Studio imports your API specification as a dependency, which means that your Mule application project now holds a reference to the specific API version that you’re implementing. Studio checks your API and notifies you when a new version is available.
```

Reply:
is "holds a reference" common terminology?

Reply:
What about, "adds" or "includes"?

Reply:
(Avoid using "This" as the subject of a sentence.)

Reply:
OK! Thanks!